Signal/src/ViewControllers/ConversationView/Cells/OWSMessageHeaderView.m

```diff
@@ -93,8 +93,8 @@ - (void)loadForDisplayWithViewItem:(ConversationViewItem *)viewItem
         [self.strokeView autoSetDimension:ALDimensionHeight toSize:strokeThickness],
 
         [self.stackView autoPinEdgeToSuperviewEdge:ALEdgeTop],
-        [self.stackView autoPinEdgeToSuperviewEdge:ALEdgeLeading withInset:conversationStyle.fullWidthGutterLeading],
-        [self.stackView autoPinEdgeToSuperviewEdge:ALEdgeTrailing withInset:conversationStyle.fullWidthGutterTrailing],
+        [self.stackView autoPinEdgeToSuperviewEdge:ALEdgeLeading],
+        [self.stackView autoPinEdgeToSuperviewEdge:ALEdgeTrailing],
```

Comment:
Aren't there corresponding changes we also need to make in the measurement logic?  I suspect we'll use it in at least one place: When determining the max width of the header text.

If so, I suggest that we move the usage of `headerGutterLeading` into `OWSMessageHeaderView` so that we gather this concern.

Reply:
You're right - I need to account for this when measuring cell width.


> If so, I suggest that we move the usage of headerGutterLeading into OWSMessageHeaderView so that we gather this concern.

Sure, I'll do that.